README.md

```diff
@@ -94,13 +94,13 @@ npx @acala-network/chopsticks@latest dry-run --config=configs/mandala.yml --prei
 
 - Run Kusama fork
   - Edit configs/kusama.yml if needed. (e.g. update the block number)
-  - `npx @acala-network/chopsticks@latest dev --config=configs/kusama.yml`
+  - `npx @acala-network/chopsticks@latest --config=configs/kusama.yml`
 
 - Setup XCM multichain
 **_NOTE:_** You can also connect multiple parachains without a relaychain
 
 ```bash
-npx @acala-network/chopsticks@latest xcm --relaychain=configs/kusama.yml --parachain=configs/karura.yml --parachain=configs/statemine.yml
+npx @acala-network/chopsticks@latest xcm -r kusama -p karura -p statemine
```

Comment:
I think we can drop `xcm` here as well. just need some logic to detect the flags

Reply:
will address on another RP